python/cuml/neighbors/nearest_neighbors_mg.pyx

```diff
@@ -192,7 +192,9 @@ class NearestNeighborsMG(NearestNeighbors):
         self._set_base_attributes(output_type=indices[0])
 
         # Specify the output return type
-        cuml.internals.set_api_output_type(self._get_output_type(queries[0]))
+        if len(queries) > 0:
```

Comment:
Since we're making this change in this PR, can we just add a simple test (or just parameterize one of the existing tests) to verify this?